testdata/routing/edge/route_edge.json

```diff
@@ -1,5 +1,5 @@
 {
-	"apiVersion": "v1",
+	"apiVersion": "route.openshift.io/v1",
```

Comment:
Will it works with 4.2/4.5/4.8 ?

Reply:
yes, it works with 4.2 and 4.8.
```
$ oc get clusterversion
NAME      VERSION                             AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.2.0-0.nightly-2021-02-22-141219   True        False         10m     Cluster version is 4.2.0-0.nightly-2021-02-22-141219

$ oc create -f auto/verification-tests/testdata/routing/edge/route_edge.json 
route.route.openshift.io/secured-edge-route created

$ oc get route secured-edge-route -oyaml
apiVersion: route.openshift.io/v1
kind: Route
```

Reply:
@lihongan Thanks for the confirm.